en/flight_modes/altitude.md

```diff
@@ -0,0 +1,55 @@
+# ALTITUDE CONTROL Flight Mode
+
+The ALTITUDE CONTROL flight mode maintains constant altitude when all remote control inputs are centered (i.e. no roll, pitch, yaw and at 50% throttle). The mode does not use GPS, and hence will not attempt to hold position against wind.

```

Comment:
I find it somewhat confusing that you mention the roll pitch inputs here. It makes it sound like altitude hold depends on them being centered.